lib/osf-components/addon/components/delete-button/template.hbs

```diff
@@ -48,20 +50,22 @@
                     }}
                 </label>
             {{/if}}
-        {{/modal.body}}
+        </modal.body>
 
-        {{#modal.footer}}
-            {{bs-button data-test-cancel-delete
-                disabled=this._deleteTask.isRunning
-                onClick=modal.close
-                defaultText=this.cancelButtonText
-            }}
-            {{bs-button data-test-confirm-delete
-                type='danger'
-                disabled=this.confirmDisabled
-                onClick=(perform this._deleteTask)
-                defaultText=this.confirmButtonText
-            }}
-        {{/modal.footer}}
-    {{/bs-modal}}
+        <modal.footer>
+            <BsButton
+                data-test-cancel-delete
+                @disabled={{this._deleteTask.isRunning}}
+                @onClick={{modal.close}}
+                @defaultText={{this.cancelButtonText}}
+            />
+            <BsButton 
+                data-test-confirm-delete
+                @type='danger'
```

Comment:
I think we decided to use double quotes in angle brackets.

Reply:
We turned this on at one point, but later turned it back off because it felt weird with curly invocation. It might make sense when we're completely converted to angle bracket invocation.